tests/test_single_topology.py

```diff
@@ -1217,3 +1246,286 @@ def test_hif2a_plot_force_constants():
+    0.6707    0.9244    0.7630 H   0  0  0  0  0  0  0  0  0  0  0  0
+    0.6707   -0.8555    0.7630 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  3  1  0  0  0  0
+  1  4  1  0  0  0  0
+  1  5  1  0  0  0  0
+  1  2  1  0  0  0  0
+M  END
+$$$$
+""",
+        removeHs=False,
+    )
+
+    mol_b = Chem.MolFromMolBlock(
+        """
+  Mrv2311 02222400253D
+
+  2  1  0  0  0  0            999 V2000
+    0.1976    0.0344    0.3479 N   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.8624    0.0345    0.6018 N   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  3  0  0  0  0
+M  END
+$$$$""",
+        removeHs=False,
+    )
+
+    core = np.array([[0, 0], [4, 1]])
+
+    ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
+    st = SingleTopology(mol_a, mol_b, core, ff)
+
+    vs_0 = st.setup_intermediate_state(0.0)
+    chiral_idxs_0 = vs_0.chiral_atom.potential.idxs
+    chiral_params_0 = vs_0.chiral_atom.params
+    assert len(chiral_idxs_0) == 4
+    assert np.sum(chiral_params_0 == DEFAULT_CHIRAL_ATOM_RESTRAINT_K) == 4
+
+    vs_1 = st.setup_intermediate_state(1.0)
+    chiral_idxs_1 = vs_1.chiral_atom.potential.idxs
+    chiral_params_1 = vs_1.chiral_atom.params
+    assert len(chiral_idxs_1) == 4
+    assert np.sum(chiral_params_1 == DEFAULT_CHIRAL_ATOM_RESTRAINT_K) == 4
+
+    np.testing.assert_array_equal(chiral_idxs_0, chiral_idxs_1)
+
+
+def test_chiral_methyl_to_water():
+    mol_a = Chem.MolFromMolBlock(
+        """
+  Mrv2311 02222411113D
+
+  5  4  0  0  0  0            999 V2000
+   -1.1951   -0.2262   -0.1811 F   0  0  0  0  0  0  0  0  0  0  0  0
+    0.1566   -0.1865    0.0446 C   0  0  0  0  0  0  0  0  0  0  0  0
+    0.4366    0.8050    0.4004 H   0  0  0  0  0  0  0  0  0  0  0  0
+    0.6863   -0.4026   -0.8832 H   0  0  0  0  0  0  0  0  0  0  0  0
+    0.4215   -0.9304    0.7960 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0  0  0  0
+  2  3  1  0  0  0  0
+  2  4  1  0  0  0  0
+  2  5  1  0  0  0  0
+M  END
+$$$$""",
+        removeHs=False,
+    )
+
+    mol_b = Chem.MolFromMolBlock(
+        """
+  Mrv2311 02222411123D
+
+  3  2  0  0  0  0            999 V2000
+   -1.1951   -0.2262   -0.1811 H   0  0  0  0  0  0  0  0  0  0  0  0
+    0.1566   -0.1865    0.0446 O   0  0  0  0  0  0  0  0  0  0  0  0
+    0.4215   -0.9304    0.7960 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0  0  0  0
+  2  3  1  0  0  0  0
+M  END
+$$$$""",
+        removeHs=False,
+    )
+
+    core = np.array([[0, 0], [1, 1], [2, 2]])
+
+    ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
+    st = SingleTopology(mol_a, mol_b, core, ff)
+
+    vs_0 = st.setup_intermediate_state(0.0)
+    chiral_idxs_0 = vs_0.chiral_atom.potential.idxs
+    chiral_params_0 = vs_0.chiral_atom.params
+    assert len(chiral_idxs_0) == 4
+    assert np.sum(chiral_params_0 == DEFAULT_CHIRAL_ATOM_RESTRAINT_K) == 4
+    vs_1 = st.setup_intermediate_state(1.0)
+
+    # chiral force constants should be on for all 4 of the 7
+    # chiral terms at lambda=1
```

Comment:
possibly adjust description to match assertion more closely

```suggestion
    # chiral force constants should be on for 4 of the 7
    # chiral terms at both lambda=0 and lambda=1
```

Reply:
willfix